Sources/DailyChallengeFeature/DailyChallengeView.swift

```diff
@@ -244,9 +244,7 @@ extension AlertState where Action == DailyChallengeAction {
         You already played today’s daily challenge. You can play the next one in \
         \(nextStartsAt, formatter: relativeFormatter).
         """),
-      primaryButton: .default(.init("OK"), action: .send(.dismissAlert)),
-      secondaryButton: nil,
-      onDismiss: .dismissAlert
```

Comment:
I've backed out of the alert API changes on `iso`. We can revisit some day, but no need to diverge from `main`.